v2/car.go

```diff
@@ -0,0 +1,70 @@
+package car
+
+import (
+	"encoding/binary"
+	"io"
+)
+
+const HeaderBytesLen uint64 = 32
```

Comment:
no need for this to be a typed constant :) and perhaps no need to export it either, as it's only useful to someone encoding/decoding CAR themselves